yaksh/ruby_stdio_evaluator.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+from __future__ import unicode_literals
+import subprocess
+import os
+from os.path import isfile
+import sys
+from contextlib import contextmanager
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+# Local imports
+from .stdio_evaluator import StdIOEvaluator
+from .file_utils import copy_files, delete_files
+from .error_messages import compare_outputs
+
+
+@contextmanager
```

Comment:
Since this is now being used in multiple places now, please add this to base evaluator and import from there.